packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,255 @@
+import { EventProcessor, Hub, Integration, Scope } from '@sentry/types';
+
+interface AWSLambdaContext {
+  getRemainingTimeInMillis: () => number;
+  callbackWaitsForEmptyEventLoop: boolean;
+  awsRequestId: string;
+  functionName: string;
+  functionVersion: string;
+  invokedFunctionArn: string;
+  logGroupName: string;
+  logStreamName: string;
+}
+
+interface Module {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
You can use `unknown` instead to make linter happy

Reply:
By using the unknown for the export .. we are getting an issue for the rapitRuntime variable hence we used any.